Packs/ServiceNow/Integrations/ServiceNowv2/ServiceNowv2_test.py

```diff
@@ -521,3 +521,33 @@ def test_get_remote_data_no_attachment(mocker):
     res = get_remote_data_command(client, args, params)
     assert res[1]['Contents'] == 'This is a comment'
     assert len(res) == 2
+
+
+def test_get_remote_data_no_entries(mocker):
+    """
+    Given:
+        -  ServiceNow client
+        -  arguments: id and LastUpdate(set to lower then the modification time).
+        -  ServiceNow ticket
+    When
+        - running get_remote_data_command.
+    Then
+        - The checked entries was not returned.
+    """
+
+    client = Client(server_url='https://server_url.com/', sc_server_url='sc_server_url', username='username',
+                    password='password', verify=False, fetch_time='fetch_time',
+                    sysparm_query='sysparm_query', sysparm_limit=10, timestamp_field='opened_at',
+                    ticket_type='incident', get_attachments=False, incident_name='description')
+
+    args = {'id': 'sys_id', 'lastUpdate': 0}
+    params = {}
+    mocker.patch.object(client, 'get', return_value=RESPONSE_TICKET_MIRROR)
+    mocker.patch.object(client, 'get_ticket_attachment_entries', return_value=RESPONSE_MIRROR_FILE_ENTRY_FROM_XSOAR)
+    mocker.patch.object(client, 'query', return_value=MIRROR_COMMENTS_RESPONSE_FROM_XSOAR)
+    mocker.patch.object(client, 'get', return_value=RESPONSE_ASSIGNMENT_GROUP)
```

Comment:
why are we patching `get` twice ?

Reply:
This are 2 different get calls to 2 different tables. So 2 different responses

Reply:
but the second one just overrides the first..
if you want to return different response for each call to `get` you can pass an array to `return_value`